src/app/access-control/epeople-registry/eperson-form/eperson-form.component.html

```diff
@@ -3,31 +3,27 @@
     <div class="col-12">
 
       @if (activeEPerson$ | async) {
-        <h1 class="border-bottom pb-2">{{messagePrefix + '.edit' | translate}}</h1>
+      <h1 class="border-bottom pb-2">{{messagePrefix + '.edit' | translate}}</h1>
```

Comment:
All these re-alignment changes should be removed, as it makes the code more difficult to read.  It's possible these occurred automatically from your IDE, but please remove any unnecessary changes to code alignment from this file.